File: php/testDBQuery.php
<?php
$init = round(microtime(true) * 1000);
//opens 100 connections to the database
for ($i = 0; $i < 100; $i++) {
	$mysqli = new mysqli("127.0.0.1", "root", "", "test");
	$res = $mysqli->query("SELECT idtestTable FROM testtable WHERE content = 'a test content with BMW'");
    $row = $res->fetch_assoc();
}

$end= round(microtime(true) * 1000);
echo ($end - $init);

File: golang/src/testDBQuery.go
package main

import (
    "database/sql"
    "fmt"
    _ "github.com/go-sql-driver/mysql"
	"time"
)

func main() {
    timestart := time.Now()

    db, err := sql.Open("mysql", "root@/test")
    if err != nil {
        panic(err.Error())
    }
    defer db.Close()
	
	//opens 100 connections to the database
    for c := 0; c < 100; c++ {
		query, err := db.Query("SELECT idtestTable FROM testtable WHERE content = 'a test content with BMW'")
		if err != nil {
			panic(err.Error())
		}

		var col1 int
		query.Next();
		err = query.Scan(&col1);
		if err != nil {
			panic(err.Error())
		}
		defer query.Close()
    }
    timeend := time.Now()

    fmt.Print(timeend.Sub(timestart))
}